webapi/Skills/ChatSkills/CopilotChatPlanner.cs

```diff
@@ -74,25 +82,69 @@ public async Task<Plan> CreatePlanAsync(string goal, ILogger logger)
             return new Plan(goal);
         }
 
-        Plan plan = this._plannerOptions?.Type == PlanType.Sequential
-               ? await new SequentialPlanner(
+        Plan plan;
+
+        switch (this._plannerOptions?.Type)
+        {
+            case PlanType.Sequential:
+                plan = await new SequentialPlanner(
                     this.Kernel,
                     new SequentialPlannerConfig
                     {
                         RelevancyThreshold = this._plannerOptions?.RelevancyThreshold,
                         // Allow plan to be created with missing functions
                         AllowMissingFunctions = this._plannerOptions?.MissingFunctionError.AllowRetries ?? false
                     }
-                ).CreatePlanAsync(goal)
-               : await new ActionPlanner(this.Kernel).CreatePlanAsync(goal);
+                ).CreatePlanAsync(goal);
+                break;
+            default:
```

Comment:
Stepwise falls within the default?

Reply:
No, this method is not called when stepwise is selected as the planner. It'll take precedence over anything else that happens in ExternalInformationSkill
![image](https://github.com/microsoft/chat-copilot/assets/125500434/dd546b57-011e-499e-915b-46618e374d0c)

Reply:
Yeah, saw that when going down the files in the PR...

Reply:
It technically doesn't create a plan as much as it executes a static one, so it didn't feel right to put it under "CreatePlanAsync". 

Will resolve this comment, feel free to re-open if you have any suggestions on how to do it differently